sdk/eventhub/Azure.Messaging.EventHubs/assets/live-tests-azure-setup.ps1

```diff
@@ -254,22 +254,22 @@ if ($resourceGroup -eq $null)
 
 if ($createResourceGroup)
 {
-    Write-Host "`t...Creaating new resource group"
+    Write-Host "`t...Creating new resource group"
     $resourceGroup = (New-AzResourceGroup -Name "$($ResourceGroupName)" -Location "$($AzureRegion)")
 }
 
 if ($resourceGroup -eq $null)
 {
-    Write-Error "Unable to locate or create the resource goup: $($ResourceGroupName)"
+    Write-Error "Unable to locate or create the resource group: $($ResourceGroupName)"

```

Comment:
I was hoping you'd edit it to `gout`

Reply:
![](https://media3.giphy.com/media/t6KmBEEzokJ8CS5bBq/giphy.gif)